### ISSUE TITLE
Alt text/long description issues in documentation's images

### ISSUE THREAD

**Issue by EricDunsworth** [cenw-wscoe/sgdc-cdts#6]
Hi,

I've noticed that the alt text/long descriptions in all of the CDTS documentation's images are either flawed or missing. IMO they'd all fail WCAG 2.0 (and 2.1) in their current state.

Here's an outline of all of them...

**[Home page](https://cenw-wscoe.github.io/sgdc-cdts/docs/index-en.html)...**
* **Image:** https://cenw-wscoe.github.io/sgdc-cdts/img/CDTS-Flow-Chart-En.png
  * **Alt text:** "CDTS version selection decision tree."
* **Issues:** 
  * Alt text doesn't mention where that image's long description can be found
  * If the list below the image is meant to act as a long description, it leaves a lot to be desired... I wouldn't consider it to be a viable alternative. For example, some of the boxes in the image's flowchart contain "Do you require a versioning system?", which isn't represented anywhere in the list.

**Other pages...**
* **[Internet page](https://cenw-wscoe.github.io/sgdc-cdts/docs/internet-en.html):**
  * **Image:** https://cenw-wscoe.github.io/sgdc-cdts/img/Cdts-page-template.png
    * **Alt text:** "CDTS page template"
* **[Intranet page](https://cenw-wscoe.github.io/sgdc-cdts/docs/intranet-en.html):**
  * **Image 1:** https://cenw-wscoe.github.io/sgdc-cdts/img/Cdts-gci-page-template.png
    * **Alt text:** "CDTS page template"
  * **Image 2:** https://cenw-wscoe.github.io/sgdc-cdts/img/Cdts-subtheme-esdc.png
    * **Alt text:** "CDTS sub theme example"
* **Issues:**
  * Alt text doesn't mention where any of the images' long descriptions can be found
  * Those images (particularly the blue boxes on the right) lack long descriptions
  * If those images were meant to be decorative, they should've used null `alt` attributes

**Comment by MaximPerry** [comment]
Thanks @EricDunsworth , I'll make sure to update that.
